queries.sql

```diff
@@ -6,4 +6,48 @@ SELECT date_of_birth from animals WHERE name = 'Agumon' OR name = 'Pikachu';
 SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutured = true;
 SELECT * FROM animals WHERE NOT name = 'Gabumon';
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE NOT name LIKE '%mon';
```

Comment:
**Changes Required**
- [ ] The most sure query here should be to update where pokemon is `NULL`. Let us try and make this change.